### PR TITLE
Add histogram registry to QA task

### PR DESCRIPTION
**Pull request by njacazio**
- Add HistogramRegistry
- Printing the name of the requested missing histogram in the registry

### CODE REVIEW

File: Framework/Core/include/Framework/HistogramRegistry.h [hunk 1]
```diff
@@ -637,7 +637,7 @@ class HistogramRegistry
         return imask(j + i);
       }
     }
-    throw runtime_error("No matching histogram found in HistogramRegistry!");
+    throw runtime_error(Form("No matching histogram %s found in HistogramRegistry!", name));
```
Comment (ktf) [inline Framework/Core/include/Framework/HistogramRegistry.h]: No need to use `Form()`, `o2::framework::runtime_error_f()` takes care of it.

```C++
o2::framework::runtime_error_f("No matching histogram %s found in HistogramRegistry!", name);
```

Comment (mario-krueger) [reply Framework/Core/include/Framework/HistogramRegistry.h]: hi, @njacazio if you agree I can include this in my next commit for the registry (which will also affect / move this function and therefore would have conflicts with your changes).

Comment (njacazio) [reply Framework/Core/include/Framework/HistogramRegistry.h]: @ktf thank you, indeed it was a temporary patch (that became permanent) ! 
@mkruegerGitHub it's up to you, I just need a way to know which plot I'm missing in the filling

Comment (mario-krueger) [reply Framework/Core/include/Framework/HistogramRegistry.h]: indeed it is a good idea to provide this information. I will add it.

Comment (njacazio) [reply Framework/Core/include/Framework/HistogramRegistry.h]: Ok, I removed the commit

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**